smf/sled-agent/non-gimlet/config.toml

```diff
@@ -64,7 +62,7 @@ swap_device_size_gb = 64
 #
 # If empty, this will be equivalent to the first result from:
 # $ dladm show-phys -p -o LINK
-# data_link = "igb0"
```

Comment:
Is this intended?

Reply:
No, that is my test setup.. how did that sneak in there.  Will fix